.travis.yml

```diff
@@ -8,22 +8,37 @@ dist: trusty
 
 language: node_js
 
-node_js:
-  - '6'
-  - '5'
-  - '4'
-  - 'iojs'
-  - '0.12'
-  - '0.11'
-  - '0.10'
-  - '0.8'
+matrix:
+  include:
+    - node_js: '6'
```

Comment:
Worth trying, but probably best done after we get the basic browser testing merged.